github/event_types.go

```diff
@@ -468,6 +468,22 @@ type OrgBlockEvent struct {
 	Installation *Installation `json:"installation,omitempty"`
 }
 
+// PackageEvent represents activity related to GitHub Packages.
+// The Webhook event name is "package".
+//
+// This event is triggered when a GitHub Package is published or updated.
+//
+// GitHub API docs: https://developer.github.com/webhooks/event-payloads/#package
+type PackageEvent struct {
+	// Action is the action that was performed.
+	// Can be "published" or "updated".
+	Action  *string       `json:"action,omitempty"`
+	Package *Package      `json:"package,omitempty"`
+	Repo    *Repository   `json:"repository,omitempty"`
+	Org     *Organization `json:"organization,omitempty"`
```

Comment:
Use full names here? Repository and Organization instead of Repo and Org. For the sake of consistency.

Reply:
This file does have mixed naming conventions where both the full name and shorthand are used; however, the shorthand (`Repo` and `Org`) versions appear to be the more popular names in this file. I think we can leave these the way they are unless we're trying to standardize to the full name going forward.

Reply:
Yes, we have not been totally consistent in this regard, so I didn't comment on these. I think that these are the two names we regularly allow to be shortened, as they are globally well known throughout the package and have very small chance of ever causing conflicts with new JSON tags that might show up in later releases.

I'm fine with leaving them as `Repo` and `Org`.